topology/University of Utah/Utah-Submit-Node/Utah-Submit-Node.yaml

```
@@ -0,0 +1,28 @@
+GroupDescription: University of Utah Center for Research and High Performance Computing
+GroupID: 528
+Production: true
+Resources:
+  OSG_US_UUTAH_SUBMIT:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID:  40a181cc3b7a43593188c9607310d35cb079bda9
+          Name: Mitchell Steinman
+      Security Contact:
+        Primary:
+          ID:  40a181cc3b7a43593188c9607310d35cb079bda9
+          Name: Mitchell Steinman
+      Site Contact:
+        Primary:
+          ID:  ddfa10c8a8500e5c31818e7c1a063b3ce5ab6a8b
+          Name: CHPC OSG Support
+    Description: Submit Node for University of Utah CHPC
+    FQDN: osg-submit.chpc.utha.edu
+    ID: x
```

```suggestion
    ID: x
```

I'm not sure how to get an ID for this.

---

You can get one by cloning the repo and running `bin/next_resource_id`.
```suggestion
    ID: 1071
```